### PR TITLE
Easings subdirectory

### LINKED ISSUE #8: Add or suggest any easings library
Make easings accessable as a timing functions from package. Functions section should be able to be separated from main package as a stand-alone library in future

### PR DESCRIPTION
**Pull request by andrewbrodko**
Easings were taken from [easings.net](https://github.com/ai/easings.net/blob/master/src/easings/easingsFunctions.ts)

- added separate build stage for easings
- adopted all easing functions with export on path `bearing/dist/easings`
- also now generating type definitions in a single rollup run

Closes #8

### CODE REVIEW

File: src/easings/index.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const PI = Math.PI;
+const c1 = 1.70158;
+const c2 = c1 * 1.525;
+const c3 = c1 + 1;
+const c4 = (2 * PI) / 3;
+const c5 = (2 * PI) / 4.5;
+
+const bounceOut = (x: number): number => {
+  const n1 = 7.5625;
+  const d1 = 2.75;
```
Comment (alexanderbrodko) [inline src/easings/index.ts:14]: Carrying will be great for this params. For example, `easingFunc = easeInBounce(7.5625, 2.75); x = easingFunc(t)`. But that way is good too

**Review by alexanderbrodko — APPROVED** [review]
So, why don't you make standalone repo for using it as a submodule?

**Comment by andrewbrodko** [comment]
> 
> 
> So, why don't you make standalone repo for using it as a submodule?

Easing functions were taken from official [easings.net repo](https://github.com/ai/easings.net). The [license](https://github.com/ai/easings.net/blob/master/LICENSE) allows us to take the parts of code with copyright notices, so at first glance there's no problems in organizing a separate package
Making easings as a submodule will definitely worth it (it allows users to install it as a package separated from main Bearing module). For example, you can look at [mui](https://github.com/mui-org/material-ui) package repo: there are `packages` folder containing mui parts, which are built separately - they all could be installed independently. Organizing such a structure may take undefined time spent on non-primary tasks at the current Bearing stage, so this will worth an issue in a future

File: src/easings/index.ts (hunk 1 above)
Comment (andrewbrodko) [reply src/easings/index.ts:14]: Carrying is a good feature. In this case, it's not worth to use carrying because of a single usage of functions

**Pull request merged**